Y2F/@optimizerFORCES/generateCInterfaceCode.m

```diff
@@ -18,21 +18,21 @@
 end
 
 % Check if FORCES solver has been generated
-if ~isdir(self.codeoptions{1}.name) && ~isdir(solverName)
+if ~isfolder(self.codeoptions{1}.name) && ~isfolder(solverName)
```

Comment:
`isfolder` was introduced in 2017b. maybe we should keep `isdir` or `exist(field, 'dir')` when older versions of MATLAB are detected (and we could form full paths to avoid checking the MATLAB path instead)

Reply:
did not know `isfolder` is that recent, will replace it by `exist(...,'dir')>0`